lawnchair/src/app/lawnchair/ui/preferences/components/README.md

```diff
@@ -0,0 +1,7 @@
+# About the `components` directory
+The `components` directory should be organized into the following:
+
+* For components for a specific UI that require many files, add it in a subdirectory. Example: `colorpreference`
+* For components that allow the user to control/select something (a list, yes/no, etc), add it in the `controls` subdirectory.
+* For components that are related to the layout itself and unrelated to the above, add it in the `layout` directory.
+* Anything else, you can add it in the current `./` directory.
```

Comment:
Should we leave this md file under sourceSet?

Reply:
Should be fine. (Not sure actually)